client/src/pages/Space/forms/CreateEntryForm/CreateEntryForm.tsx

```diff
@@ -18,10 +15,10 @@ interface Incoming {
   setPosts: Function;
 }
 
-function CreateEntryForm(props: Incoming) {
-  const URL = process.env.REACT_APP_API + '/posts';
+function CreateEntryForm({setOpened, space_id, user_id, setPosts}: Incoming) {
   const [richTextValue, setRichTextValue] = useState('');
   const [title, setTitle] = useState('');
+  // todo what about the tags that I created?
```

Comment:
This is a funny one, you can create custom tags when you create a post but you CANNOT reuse them when you create a new post :D